src/chronometer.js

```diff
@@ -1,33 +1,48 @@
 class Chronometer {
   constructor() {
-    // ... your code goes here
+    this.currentTime = 0;
+    this.intervalId = null;
   }
 
   start(printTimeCallback) {
-    // ... your code goes here
-  }
-
+  this.intervalId = setInterval (() => {
+  this.currentTime++
+  if (printTimeCallback) return printTimeCallback()
+  }, 1000);
+}
   getMinutes() {
-    // ... your code goes here
+    if (!this.currentTime) return 0
```

Comment:
You don't need this if statement here.
`return Math.floor(this.currentTime / 60)` is correct!